.gitignore

```diff
@@ -4,3 +4,4 @@
 build
 dist
 docs/_build
+.idea/
```

Comment:
Heh, noticed that during the PR and was wanting to change it anyway.  Good call.